addons/hr/models/mail_activity.py

```diff
@@ -0,0 +1,6 @@
+from odoo import fields, models
+
+class MailActivity(models.Model):
+    _inherit = 'mail.activity'
+
+    plan_id = fields.Many2one('mail.activity.plan', string='Plan')
```

Comment:
Move this part into ``mail`` directly, add python tests as well to ensure the plan_id is correctly set on activity schedule.